### ISSUE TITLE
bug(select): cannot close non-native select menu with Talkback and iOS voiceover without selecting an option

### ISSUE THREAD

**Issue by zarend** [angular/components#24810]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

For the non-native select, there is no way to close the options menu on mobile screen readers without selection an option. This affects both the material select and MDC-based select.

### Reproduction

Steps to reproduce:
1. with iOS voiceover or Talkback screenreader enabled, navigate to https://material.angular.io/components/select/overview
2. Navigate to "Basic mat-select" demo
3. Click on "Favorite Food"
4. (Menu appears with "Steak", "Pizza" and "Tacos")

### Expected Behavior

There should be a way to close the menu without selecting an option.

### Actual Behavior

There is no way to close the menu without selecting an option.

### Environment

screen readers: iOS Voiceover, Android Talkback

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
